docs/templates.md

```diff
@@ -148,6 +148,14 @@ defined.
 * `.remove_suffix(needle: Template) -> String`: Removes the passed suffix, if present
 * `.substr(start: Integer, end: Integer) -> String`: Extract substring. Negative values count from the end.
 
+#### String literals
+
+String literals must be surrounded by double quotes (`"`). The following escape
+sequences starting with a backslash have their usual meaning: `\"`, `\\`, `\n`,
+`\r`, `\t`, `\0`. Other escape sequences are not supported. Any UTF-8 characters
+are allowed inside a string literal, with two exceptions: unescaped `"`-s and
+uses of `\` that don't form a valid escape sequence.
+
```

Comment:
nit: perhaps, the statements about invalid escape sequence can be combined? Something like "The following escape sequence [..] are supported: [..]. Unsupported escape sequence is an error."

Reply:
I tried to achieve something like that, but kept finding pedantic issues with that kind of wording. In particular, I didn't want to define what an "escape sequence" is (as opposed to a "valid escape sequence", which I enumerated all of).

So, I think I'll keep it as is. Perhaps somebody will think of a better way or I'll feel less pedantic in the future.